cdap-docs/included-applications/source/etl/plugins/transforms/script.rst

```diff
@@ -72,6 +76,15 @@ schema is the same as the input schema.
           {\"name\":\"tax\",\"type\":\"double\"},
           {\"name\":\"total\",\"type\":\"double\"}
         ]
+      }",
+      "lookup": "{
+        \"purchases\":{
+          \"type\":\"DATASET\",
+          \"properties\":{
+            \"dataset_argument1\":\"foo\",
+            \"dataset_argument2\":\"bar\"
```

Comment:
I think this and the documentation task below (for ``getLookup()``) should be a new PR, tracked by a new issue. Can you create one?